ci/dockerfiles/autoscaler-devbox/Dockerfile

```diff
@@ -0,0 +1,18 @@
+FROM jetpackio/devbox@sha256:430afb5a5c9b4ec9f56776e8fcb82fe67fa5fabba3ba67ebb1c13b17d0e8de8d
```

Comment:
Have you tested if the CI-pipeline required stuff like `make acceptance-tests`, `make test` etc. works with this docker image?